packages/@lwc/engine-core/src/framework/check-version-mismatch.ts

```diff
@@ -38,7 +38,7 @@ export function checkVersionMismatch(
 ) {
     const versionMatcher = func.toString().match(LWC_VERSION_COMMENT_REGEX);
     if (!isNull(versionMatcher) && !warned) {
-        if (process.env.SKIP_LWC_VERSION_MISMATCH_CHECK === 'true') {
+        if (!process.env.IS_BROWSER || process.env.SKIP_LWC_VERSION_MISMATCH_CHECK === 'true') {
```

Comment:
```suggestion
        if (typeof process === 'object' && process.env.SKIP_LWC_VERSION_MISMATCH_CHECK === 'true') {
```
And we've come full circle. 😓

Reply:
![fullcircle](https://github.com/user-attachments/assets/20b5bd46-baf3-4d4d-8971-666d7449e832)